CHANGELOG.rst

```diff
@@ -16,10 +16,20 @@ The format is based on `Keep a Changelog`_.
   - **Fixed**: for any bug fixes.
   - **Security**: in case of vulnerabilities.
 
-
 `UNRELEASED`_
 =============
 
+
+`0.7.2`_ - 2019-03-05
+=====================
```

Comment:
This needs to be removed this isn't a 0.7.2 release and we also didn't push a 0.7.2 release on Tuesday.

Reply:
Thanks for the feedback. I'll get that updated

Reply:
@mtreinish I have updated the changelog to reflect your feedback.